Mise à jour complète du projet

cristian-aldea opened the pull request:
Ce pull request va ajouter quelques commits essentiels que j'ai fait pour le GameManager et les ennemis. Allez voir les descriptions des commits pour plus de détails.

Commit:
Update 15/03/2017

Changement
-Commencement du GameManager(système de sauvegarde, système de gestion
des niveau,etc...)
-Création du GameSave pour sauvegarder le progrès de Spellington

Commit:
Small update GameSave

Changements:
Ajout de quelques éléments dans le GameSave

Commit:
Resolution de conflits

Commit:
Résolution de conflits

Commit:
Merge remote-tracking branch 'refs/remotes/origin/master' into Game_Manager_Creation

Commit:
creation de spell passif pour les résistances

Commit:
ajout des différente cartes fonctionnelle de tuto et de cachot

+ 12 cartes

Commit:
Merge remote-tracking branch 'refs/remotes/origin/master' into système-de-sort-Celtis

Commit:
création des sorts d'explosion + passif(soin et saut et resistance)  + destruction de classe inutile + map + dégat de living vérifie les rés

# Conflicts:
#	Spellington/src/ca/qc/bdeb/info204/spellington/calculations/Spell.java

Commit:
Code des ennemies et collision pour les projectiles

Changements :
-Collision pour les projectiles
-Début de la structure pour les ennemies
-Changement du font du incantationText dans  le hud

Commit:
changement de toutes les images de spellington pour une taille de 300px

Commit:
plus d'animation à la bonne taille!! correction de la taille de spellington

2017-04-08 celtis

Commit:
gestion des animation pour les sorts (y compris passif!) et point final pour tout les sorts passif sauf les immunitées

Commit:
Petit changements dans le main menu et MenuItem

-Mise à jour du menu avec les boutons "load" et "new" game
-Changements mineurs de menuitem

Commit:
Merge remote-tracking branch 'refs/remotes/origin/système-de-sort-Celtis' into Game_Manager_Creation

Commit:
Changements majeurs au GameManager

Commit:
Complétion du système de gestion du jeu

Changements:
-Save files fonctionnels
-On a le choix entre faire un nouveau jeu ou charger un jeu déjà
commencé
- Modification de la méthode qui extrait tous les infos du TiledMap pour
inclure tous les Events. Par exemple, maintenant, Spellington va
commencer la chambre dans le spawn point de la map comme il le faut.
-Progrès sur les classes Enemy. Ce n'est pas encore fonctionnel mais j'y
suis presque
-Correction de bugs dans l'animation de Spellington.
-Ajout d'un nouveau BasicGameState pour choisir les niveaux qui
s'appelle LevelSelectionState
-Changements mineurs à MenuItem pour répondre à quelques besoin
essentiels dans la fonctionnalité des boutons.

Commit:
Update from Grimoire

Mise à jour de ma branche avec celle de tarik et résolution des nombreux
conflits dans le code.

TheGreatMarkus commented:
Après que vous avez accepté ce pull request. Je vous prie de delete vos vielles branches et de travailler à partir de nouvelles.

Commit:
Changements des noms de fichiers

Changements :
-Changements de noms de certains fichiers pour que tous soit en anglais
-Changements de fichiers .tmx pour refl/ter les changements apportés aux
noms et lieux des fichiers tuiles
